.github/workflows/maven.yml

```diff
@@ -32,5 +32,7 @@ jobs:
       with:
         distribution: 'temurin'
         java-version: ${{ matrix.java }}
+    - name: compile with passing checkers
+      run: mvn -V apache-rat:check -Prungood compile --file pom.xml
```

Comment:
`rungood` doesn't convey what this is about.  I would name it "suceedingTypecheckers" or "typecheckForbidErrors" or some better name than I am thinking of right now.